Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -56,17 +65,39 @@ FuzzingTarget GetFuzzingTarget() {
     LOG_WARN("No value provided for FUZZING_TARGET environment variable.");
     return FuzzingTarget::kNone;
   }
-  if (fuzzing_target == "NONE") {
+  // Find the FuzzingTarget enum value that corresponds to the provided
+  // FUZZING_TARGET string value.
+  std::unordered_map<std::string, FuzzingTarget>::const_iterator t =
+      fuzzing_target_names.find(fuzzing_target);
+  if (t == fuzzing_target_names.end()) {
+    // Get all avilable keys in the map and put them in a string.
+    std::vector<std::string> all_keys;
+    for (std::unordered_map<std::string, FuzzingTarget>::iterator it = fuzzing_target_names.begin();
+         it != fuzzing_target_names.end(); it++) {
+      all_keys.push_back(it->first);
+    }
+    // Print error message with all available targets, which must be enclosed in
+    // curly brackets and separated by spaces.
```

Comment:
First, I'd expand on this slightly. In particular, I'd recommend mentioning that the fuzzing_travis.sh script parses this output, and it therefore needs to remain stable.

Secondly, I don't like this. :)  But I don't have a better suggestion. :(